src/main/java/picard/util/IntervalListTools.java

```diff
@@ -271,26 +274,25 @@
             "by any of the input intervals. Will merge abutting intervals first. Output will be sorted.", optional = true)
     public boolean INVERT = false;
 
-    @Argument(doc = "What value (if anything) to output to stdout (for scripting)")
-    public Output OUTPUT_VALUE = Output.NONE;
+    @Argument(doc = "What value to output to COUNT_OUTPUT file (for scripting)")
```

Comment:
add a commandlinevalidation that checks that COUNT_OUTPUT != null iff OUTPUT_VALUE != None

Reply:
and also validate that COUNT_OUTPUT is writable